AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -19,7 +19,7 @@ define([
             buyCreditsUrl: 'https://stock.adobe.com/',
             mediaGallerySelector: '.media-gallery-modal:has(#search_adobe_stock)',
             adobeStockModalSelector: '#adobe-stock-images-search-modal',
-            tabImagesLimit: 4,
+            tabImagesLimit: 5,
```

Comment:
That's a bit confusing.

I'd better kept `tabImagesLimit` value as it is (showing only 4 images in the tab) but requested `tabImagesLimit + 1` images from the backend